SS_write_ssnew.tpl

```diff
@@ -1537,10 +1537,40 @@ FUNCTION void write_nucontrol()
   NuFore << "# where none and simple require no input after this line; simple sets forecast F same as end year F" << endl;
   NuFore << N_Fcast_Yrs << " # N forecast years " << endl;
   NuFore << Fcast_Flevel << " # Fmult (only used for Do_Forecast==5) such that apical_F(f)=Fmult*relF(f)" << endl;
-  NuFore << "#_Fcast_years:  beg_selex, end_selex, beg_relF, end_relF, beg_mean recruits, end_recruits  (enter actual year, or values of 0 or -integer to be rel. endyr)" << endl
+  anystring = "";
+  if(Fcast_yr_rd(1) != -12345)
+  {   //  write in old format
+    NuFore << "#_Fcast_years for averaging:  beg_selex, end_selex, beg_relF, end_relF, beg_mean recruits, end_recruits  (enter actual year, or values of 0 or -integer to be rel. endyr)" << endl
```

Comment:
In other places in the revised code, we use the word "mean" but here we use "average". I would suggest using one or the other for consistency.